lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +206,41 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self._seed_urls = config.get_seed_urls()
+        self._config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        href = article_bs.get('href')
+        if isinstance(href, str) \
+                and href.startswith('https://chelny-izvest.ru/news/') \
+                and href.count('/') == 5:
+            return href
+        return ''
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        for seed_url in self._seed_urls:
+            response = make_request(seed_url, self._config)
```

Comment:
how do you check success or fail of request? address

Reply:
done